README.md

```diff
@@ -4,12 +4,16 @@ Generates a basic analysis of a CSV to an XLSX workbook.
 ```
 usage: csv_summary [-h] [-o OUTPUT_PATH]
                    [--category-threshold CATEGORY_THRESHOLD]
-                   [--date-format DATE_FORMAT] [-i IGNORE_VALUE]
-                   [-s NUM_SAMPLES]
-                   csv_path
+                   [--date-format DATE_FORMAT]
+                   [--date-time-format DATE_TIME_FORMAT] [-i IGNORE_VALUE]
+                   [--sheet-name SHEET_NAME] [-s NUM_SAMPLES]
+                   input
 
 positional arguments:
-  csv_path
+  input                 CSV or XSL input containing data. If XLS/XLSX, will
```

Comment:
XSLX is far more common, include both, ever just XSL on its own. OR do you mean .xls* for the many Excel formats?

Reply:
`XLS` is obviously a typo 🤦 I will trim down the docs to only mention `XLSX`, since it's the most common form.